README.md

```diff
@@ -59,13 +59,12 @@ This allows for this plugin to also lint your configuration files inside the .st
 
 > For more info on why this line is required in the .eslintignore file, check this [ESLint documentation](https://eslint.org/docs/latest/use/configure/ignore-deprecated#:~:text=In%20addition%20to,contents%20are%20ignored).
```

Comment:
nit: to make sure that the legacy/flat config snippets stay side by side,

- I would move this callout and the paragraph before, to place them after the flat config example
- I would rework the wording to say "For more info on why this line is required in the .eslintignore file or the `ignores` object, ..."